File: Fortran/grace.f90
#define A(x,y) WRITE(2,y,advance="no"),34,34,34,34;WRITE(2,x,advance="no"),34,34,39,y,39,39,x,39
#define B() PROGRAM Grace!E
#define C() OPEN(unit=2,file="Grace_kid.f90")
B()
CHARACTER*185::s='("#define A(x,y) WRITE(2,y,advance=",1A1,"no",1A1,"),34,34,34,34;WRITE(2,x,advance=",1A1,"no",1A1,"),34,34,39,y,39,39,x,39",/,"#define B() PROGRAM Grace!E",/,"#define C() OPEN(unit=2,")'
CHARACTER*130::t='("file=",1A1,"Grace_kid.f90",1A1,")",/,"B()",/,"CHARACTER*185::s=",1A1,1A,1A1,/,"CHARACTER*130::t=",1A1,1A,1A1,/,"C();A(t,s);END")'
C();A(t,s);END

File: C/grace.c
#include <stdio.h>
#include <fcntl.h>
/*
	a
*/
#define WR(x, s) dprintf(x, s, 10, 10, 10, 10, 10, 10, 34, 34, 10, 34, s, 34, 10, 10)
#define	OP() ((fd = open("Grace_kid.c", O_WRONLY | O_CREAT | O_TRUNC)) > 0 ? fd : 1)
#define FT() int main(){int fd;return(WR(OP(), "#include <stdio.h>%c#include <fcntl.h>%c/*%c	a%c*/%c#define WR(x, s) dprintf(x, s, 10, 10, 10, 10, 10, 10, 34, 34, 10, 34, s, 34, 10, 10)%c#define	OP() ((fd = open(%cGrace_kid.c%c, O_WRONLY | O_CREAT | O_TRUNC)) > 0 ? fd : 1)%c#define FT() int main(){int fd;return(WR(OP(), %c%s%c));}%cFT()%c"));}
FT()
